File: FDP/T5_A6_PinedoBerumenClaraRuby/Ejercicio10_Imprimir_Cada_Palabra.py
'''
Created on 4 dic. 2019

@author: Ruby
'''
print("Programa que lee una frase e imprime cada palabra y su\n",
      "tamanio en lineas diferentes\n")

frase = input("ingresa una frase: ")

inicio = 0

for i in range(len(frase)):
    if(i==" "):
        palabra = frase[inicio:i]
        
        print(f"La palabra {palabra} tiene {len(palabra)} caracteres" )
        
        inicio = i+1

File: FDP/T3_A8_PinedoBerumenClaraRuby/src/Ejercicio10_Imprimir_Cada_Palabra.java
import java.util.Scanner;

public class Ejercicio10_Imprimir_Cada_Palabra {

	public static void main(String[] args) {
		
		Scanner entrada = new Scanner(System.in);
		System.out.println("==Programa que lee una frase e imprime cada palabra y su \r\n"
				+ "tamaño en líneas diferentes==");
		System.out.println();
		
		System.out.println("Ingresa una frase: ");
		String frase = entrada.nextLine() + " ";
		
		int inicio = 0;
		
		for(int i=0; i<frase.length(); i++ ) {
			char ascii = frase.charAt(i);
			if ( ascii == ' ') {
				
				String palabra = frase.substring(inicio, i);
				
				System.out.println("La palbra "+palabra+" tiene "+ palabra.length()+" caracteres");
				
				inicio = i+1;
				
				
			}//if
			
			
			
		}//for
		
		
	

	
	}

}
